### PR TITLE
doc: spreg links

### PR DESCRIPTION
**Pull request by sjsrey**
Addresses #85

### CODE REVIEW

**Comment by weikang9009** [comment]
Can you create a PR with changes to source rst files on pysal.site? https://github.com/pysal/pysal.site/pulls

**Comment by sjsrey** [comment]
> Can you create a PR with changes to source rst files on pysal.site? https://github.com/pysal/pysal.site/pulls

I can't. I am hitting the same errors from before and I've given up on trying to keep the two repos integrated.

At the dev meeting we decided to start to move to something more simplified - so for now I think we should just update the pysal.github.io sources directly.

File: about.html [hunk 1]
```diff
@@ -92,7 +91,66 @@
 
 <div class="container">
   <div class="row">
-    <div class="col-md-12 content">
+      <div class="col-md-3">
+        <div id="sidebar" class="bs-sidenav" role="complementary"><ul>
+<li><a class="reference internal" href="#">About PySAL</a><ul>
```
Comment (weikang9009) [inline about.html]: I think the sidebar is not quite pretty under the current bootstrap html theme.

**Comment by weikang9009** [comment]
resolved by https://github.com/pysal/pysal.github.io/pull/89

**Pull request closed without merging**